Assets/Scripts/Interaction/Interactable.cs

```diff
@@ -13,6 +13,15 @@ public abstract class Interactable : MonoBehaviour
 {
     public string displayText;
     public string displaySubtext;
+    public bool ignoreTagCheck = false;
```

Comment:
what does this field do?

Reply:
This field can be checked, if you want to ignore the warning for a GameObject in the scene - for whatever reason. It was just easy to implement and might be helpful